src/admin/settings/settings.scss

```diff
@@ -0,0 +1,118 @@
+.components{
+    float: left;
+    position: relative;
+    min-height: 1px;
+    padding-right: 15px;
+    padding-left: 15px;
+}
+.importexport{
+    float: left;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    min-height: 20px;
+}
+.form-group{
+    >.settings-box-content{
+       border-radius: 4px;
+       border: 1px solid #D4E1F9;
+       padding: 0;
+       @include linear-gradient(to bottom, #eff8fd 0%, #F6FBFF 6%, #F6FBFF 16%, #F3FBFE 16%, #F0F9FE 37%, #F0F8FB 39%, #E8F5FB 57%, #E8F2FB 67%, #dbedf7 100%);
+       >.settings-box-header{
+           border-radius: 2px;
+           font-weight: bold;
+           line-height: 20px;
+           font-size: 15px;
+           padding: 5px 10px;
+           color: #c2eafe;
+           background: #144573;
+       }
+       >.settings-form{
+            border: transparent;
+            border-left:1px solid #D4E1F9;
+            padding: 10px 0 0 10px;
+            >legend{
+                color: #224477;
```

Comment:
Try not to make this level of a change to a legend -- we want styles across all pages to mostly look the same except for elements that are custom.

(I will make sure this is clear in the UI-Migration instructions, as it occurred to me that it might not be)